pkg/apis/apis/v1alpha1/types_apiexport_test.go

```diff
@@ -115,9 +115,41 @@ func TestAPIExportPermissionClaimCELValidation(t *testing.T) {
 				"openAPIV3Schema.properties.spec.properties.permissionClaims.items: Invalid value: \"object\": either \"all\" or \"resourceSelector\" must be set",
 			},
 		},
+		{
+			name: "logicalcluster invalid",
+			current: map[string]interface{}{
+				"group":    "tenancy.kcp.dev",
+				"resource": "logicalclusters",
+				"all":      true,
+			},
+			wantErrs: []string{
+				"openAPIV3Schema.properties.spec.properties.permissionClaims.items: Invalid value: \"object\": logicalclusters cannot be claimed",
+			},
+		},
+		{
+			name: "logicalcluster invalid with empty identityHash",
+			current: map[string]interface{}{
+				"group":        "tenancy.kcp.dev",
+				"resource":     "logicalclusters",
+				"identityHash": "",
+				"all":          true,
+			},
+			wantErrs: []string{
+				"openAPIV3Schema.properties.spec.properties.permissionClaims.items: Invalid value: \"object\": logicalclusters cannot be claimed",
+			},
+		},
+		{
+			name: "logicalcluster invalid with identityHash",
```

Comment:
This says invalid but there's no wantErrs?

Reply:
wrong title. Fixed.